clients/fides-js/__tests__/lib/cookie.test.ts

```diff
@@ -484,165 +444,6 @@ describe("updateExperienceFromCookieConsent", () => {
       ]);
     });
   });
-
-  describe("tcf", () => {
```

Comment:
Had to remove these tests because `updateExperienceFromCookieConsent` actually needs to use the new `updateExperienceFromCookieConsentTcf` method.

I had some issues with jest / preact exporting it from `fides-tcf.ts`, but decided to remove it as a) this is already covered by our e2e tests and b) `updateExperienceFromCookieConsentTcf` exists in `fides-tcf.ts` and is therefore not a candidate for a `cookie.ts` unit test.